nested/mssql.json

```diff
@@ -0,0 +1,63 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01-preview/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "moodleCommon": {
+            "metadata": {
+                "description": "Common Moodle values"
+            },
+            "type": "object"
+        }
+    },
+    "resources": [
+        {
+            "apiVersion": "2015-05-01-preview",
+            "kind": "",
+            "location": "[resourceGroup().location]",
+            "name": "[parameters('moodleCommon').serverName]",
+            "properties": {
+                "administratorLogin": "[parameters('moodleCommon').dbLogin]",
+                "administratorLoginPassword": "[parameters('moodleCommon').dbLoginPassword]",
+		"sslEnforcement": "[parameters('moodleCommon').sslEnforcement]",
```

Comment:
Minor: Indentation, but I also don't see this property in [the published REST API](https://docs.microsoft.com/en-us/rest/api/sql/servers/createorupdate). Is this new in the preview API version you used? Could you point me to the preview API version references?

Reply:
Property removed, not sure where I got it from, perhaps a sample template somewhere.